view/frontend/templates/js/replacejs.phtml

```diff
@@ -1707,13 +1707,26 @@ if (!$block->isSaveCartInSections()) { ?>
         var cartBarrier;
         var countConfigureCalls = 0;
 
+        var boltCheckoutConfigure = function(cart, hints, callback, parameters = undefined) {
+            var callConfigure = function() {
+                BC = BoltCheckout.configure(cart, hints, callback, parameters);
+            }
+            // Check if BoltCheckout is defined (connect.js executed).
+            // If not, postpone processing until it is
+            if (!window.BoltCheckout) {
+                whenDefined(window, 'BoltCheckout', callConfigure);
+                return;
+            }
+            callConfigure();
```

Comment:
Why refactor out one line into `callConfigure`?

Reply:
Not sure I understand the question. Could you explain?

Reply:
Nevermind, I missed the call in `whenDefined` >__> 
My bad.